### ISSUE TITLE
Restore function removed from gl-admin

### ISSUE THREAD

**Issue by mryber** [globaleaks/GlobaLeaks#2485]
**Current behavior**

After upgrading to 3.6.0, the Restore function from gl-admin has been removed. I have checked documentation and can see that the procedure for backup and restore has changed, but it is not described in the ChangeLog.

**Expected behavior**

N/A

**Steps to reproduce the problem or feature illustration**

Upgrade from version 3.5.8 to 3.6.0

**What is the motivation or use case for changing the behavior?**

Would like to know the reason for removing the Restore functionality, as we now have to write our own Backup/Restore scripts, which complicates thing when working with external partners.

**GlobaLeaks version:**

v3.6.0+

**Browser:**

N/A - this is a commandline thing.

**Server Operating System and Version (if applicable):**
 
Ubuntu 18.04.1 LTS

**Client Operating System and Version (if applicable):**

N/A - we are running on the server.

**Comment by evilaliv3** [comment]
Thank you @mryber, the functionality has been removed because it requires new implementation following the refactor for automatic backups implemented in 3.6.0.

The restore functionality will be sooner or later exposed also via the web interface but it requires some development.

**Comment by evilaliv3** [comment]
To facilitate your activities we will try to reimplement the restore command for the next release.

It is enough to have it as it was or would you need something else?

**Comment by mryber** [comment]
Okay - I can see that the restore process is rather simple, by removing /var/globaleaks and untar the archive and start the service - on that account I came across that the upgrade will place a backup in /var/globaleaks/backups, but a restore will delete that backups directory as well, so for now I created a backup script to place backups in fx. /var/gl-backups, to keep them out of the work-directory.

Backup/Restore via the web interface would be a nice to come feature. You don't need to reimplement the backup feature, though I can see it's a simpe piece of Python code, in my case I am developing site on one system, and I can supply hosting provider with a guide on how to restore site to live system. I figured out to change the hostname variable in the sqlite3 database, so admin will be able to login afterwards as well.
